haiku-apps/einsteinium/einsteinium-1.2.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Monitor applications and system services"
+DESCRIPTION="Einsteinium provides smarter monitoring of applications and \
+system services for Haiku. It can restart applications and system services \
+that were quit or crashed, gather statistics on application usage and \
+provides customizable ranked lists of applications."
+HOMEPAGE="https://sourceforge.net/projects/esforhaiku/"
+COPYRIGHT="2010-2016 Brian Hill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/esforhaiku/files/Einsteinium-${portVersion}_source.zip/download"
+CHECKSUM_SHA256="9025e10a4aa90d3baee165e7b11781df3ba157073aa5944f282421573124a18a"
+SOURCE_DIR="Einsteinium-${portVersion}_source"
+
+ARCHITECTURES="x86_gcc2 !x86"
+
+PROVIDES="
+	einsteinium = $portVersion
+	app:Einsteinium = $portVersion
+	cmd:einsteinium_engine = $portVersion
+	cmd:einsteinium_daemon = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsqlite3
+	lib:libxml2
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsqlite3
+	devel:libxml2
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+USER_SETTINGS_FILES="settings/Einsteinium directory"
+
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/einsteinium_hpkg_postinstall.sh"
+
+BUILD()
+{
+	cd src
+	for f in Engine Daemon; do
+		pushd $f
+		make $jobArgs OBJ_DIR=.
+		popd
+	done
+	for f in Preferences Launcher; do
+		pushd $f
+		make $jobArgs
+		popd
+	done
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Einsteinium/EngineSubscriberKit
+	mkdir -p $binDir
+	mkdir -p $dataDir/Einsteinium
+	mkdir -p $dataDir/user_launch
+	mkdir -p $preferencesDir
+	mkdir -p $postInstallDir
+
+	# copy documentation
+	cp -af docs/ReadMe docs/License $appsDir/Einsteinium
+	cp -af "docs/Engine Subscriber's Development Guide" $appsDir/Einsteinium/EngineSubscriberKit
+
+	# post install script
+	cp -af scripts/einsteinium_hpkg_postinstall.sh $postInstallDir
+	chmod +x $postInstallDir/einsteinium_hpkg_postinstall.sh
+	cp -af src/Icons/notify_icon $dataDir/Einsteinium
+
+	# launch settings
+	cp -af scripts/einsteinium_userlaunch $dataDir/user_launch/Einsteinium
+
+	# copy executables
+	cd src
+	cp -af Engine/einsteinium_engine Daemon/einsteinium_daemon $binDir
+	cp -af Preferences/Einsteinium_Preferences $preferencesDir/Einsteinium
+	cp -af Launcher/Einsteinium_Launcher $appsDir/Einsteinium/Einsteinium\ Launcher
+
+	# create the Subscriber Example Kit
+	cp -af Engine/SubscriberExample/makefile Engine/SubscriberExample/subscriber_example \
+Engine/SubscriberExample/subscriber_example.rsrc Engine/SubscriberExample/SubscriberExample.cpp \
+Engine/EngineSubscriber.cpp Engine/EngineSubscriber.h $appsDir/Einsteinium/EngineSubscriberKit
+
```

Comment:
Would be nicer if those lines were indented twice. Possibly also only have one file per line, as with the paths they get pretty long. Or copy first the two files from Engine/ and the cd into Engine/SubscriberExample. Or cp Engine/SubscriberExample/* .